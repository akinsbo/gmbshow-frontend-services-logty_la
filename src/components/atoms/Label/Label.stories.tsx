import { withKnobs } from '@storybook/addon-knobs'
import { storiesOf } from '@storybook/react'
import * as React from 'react'
// import Label from './Label'
import Button from '@material-ui/core/Button';


storiesOf('atoms|Label', module)
  .addDecorator(withKnobs)
  .add('with text', () => <Button/>, { notes: 'Simple label component' })
  // .add('with some emoji', () => (
  //   <Label>
  //     <span role="img" aria-label="so cool">
  //       😀 😎 👍 💯
  //     </span>
  //   </Label>
  // ),

