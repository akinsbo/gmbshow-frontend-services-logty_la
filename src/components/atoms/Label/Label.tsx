import * as React from "react"

const Label = () => (
    <label>Label</label>
)

export default Label