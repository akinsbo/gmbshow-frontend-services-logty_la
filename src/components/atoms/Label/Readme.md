# Label

This component describes the style of Labels in this stylesheet
