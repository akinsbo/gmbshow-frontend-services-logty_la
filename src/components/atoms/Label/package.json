{
    "name": "Label",
    "version": "1.0.0",
    "description": "Label component",
    "main": "Label.tsx",
    "scripts": {
        "test": "jest"
    },
    "repository": {
        "type": "git",
        "url": "https://github.com/akinsbo/gmbshow-frontend-services-logty_la/src/components/atoms/Label"
    },
    "keywords": [
        "header",
        "component"
    ],
    "author": "Olaolu Akinsete",
    "license": "ISC"
}