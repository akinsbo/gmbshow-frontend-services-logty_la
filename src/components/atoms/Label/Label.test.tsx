
import * as React from "react"
import { render } from '@testing-library/react'

import Label from "./Label";

describe("Label", () => {
  it("renders correctly", () => {
    const { container } = render(
      <Label><h1>Word</h1></Label>
    )
    expect(container).toMatchSnapshot()
  })
})