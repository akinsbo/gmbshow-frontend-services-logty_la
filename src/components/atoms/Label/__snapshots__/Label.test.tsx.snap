// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label renders correctly 1`] = `
<div>
  <label>
    <h1>
      Word
    </h1>
  </label>
</div>
`;
